includes/class-database-cleanup.php

```
@@ -587,15 +587,23 @@ public static function clean_all() {
-			$results = array();
+			$results       = array();
+			$total_deleted = 0;
+
 			foreach ( $methods as $key => $method ) {
 				if ( 'revisions' === $key ) {
-					$results[ $key ] = self::invoke_cleanup_method( $method, 30, 5 );
+					$res = self::invoke_cleanup_method( $method, 30, 5 );
 				} else {
-					$results[ $key ] = self::invoke_cleanup_method( $method );
+					$res = self::invoke_cleanup_method( $method );
+				}
```

PHPMD (2.15.0)

[error] The method clean_all uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)